uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/controllers/casetype.extended.controller.spec.js

```diff
@@ -43,7 +43,50 @@
       });
 
       it('initialises the parent controller with activity types having component labels', function () {
-        expect(parentControllerSpy).toHaveBeenCalledWith(expectedApiCalls);
+        expect(parentControllerSpy).toHaveBeenCalledWith(jasmine.objectContaining(expectedApiCalls));
+      });
+
+      describe('default relationship options', function () {
+        var expectedRelationshipOptions;
+
+        beforeEach(function () {
```

Comment:
This beforeEach is pretty difficult to follow. Could we send some code out into a function? I think if we move this block https://github.com/compucorp/civihr-tasks-assignments/pull/408/files#diff-cb6f7866f0ad87f9a9f0a4c3669b56c0R60 to a separate func and call it accordingly, that may work.